portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -19,14 +19,50 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
+
+import org.json.simple.JSONObject;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
-@WebServlet("/data")
+@WebServlet("/comment")
 public class DataServlet extends HttpServlet {
+  ArrayList<String> allComments = new ArrayList<>();
```

Comment:
I believe this is your temp storage for comments, right? Usually the HTTP handler is state-free, which means it doesn't store any user-session data in itself, as we usually have many replicas of the handler, every time you request the server, the request will be handled by different handlers, which means your data is only available in one replica, but not others.